apps/mn-original-populasjon/src/main/java/no/nav/registre/testnorge/originalpopulasjon/domain/Adresse.java

```diff
@@ -0,0 +1,33 @@
+package no.nav.registre.testnorge.originalpopulasjon.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+
+import no.nav.registre.testnorge.libs.dto.person.v1.AdresseDTO;
+
+@Builder
+@AllArgsConstructor
```

Comment:
Trenger ikke @AllArgsConstructor hvis du har @Value

Reply:
Får feil hvis jeg fjerner AllArgsConstructor. Den ser rett til konstruktøren under som tar inn AdresseDTO. Noen magisk @ som fikser det?

`java: constructor Adresse in class no.nav.registre.testnorge.originalpopulasjon.domain.Adresse cannot be applied to given types;
  required: no.nav.registre.testnorge.libs.dto.person.v1.AdresseDTO
  found: java.lang.String,java.lang.String,java.lang.String,java.lang.String
  reason: actual and formal argument lists differ in length`